### ISSUE TITLE
" Si je fais deux fois la même recherche, je n'ai pas le même nombre de résutlats" 

### ISSUE THREAD

**Issue by Eliselalique** [entrepreneur-interet-general/CIS-front#197]
Problème remonté par Emmanuel
-> essayer de reproduire le bug pour vérifier la pertinence de l'issue

**Comment by DavidBruant** [comment]
> essayer de reproduire le bug

Oui, j'aurais besoin d'un exemple spécifique (quels mots-clefs, quels filtres) pour pouvoir creuser

**Comment by DavidBruant** [comment]
S'agissait-il de #203 ou d'un autre problème ?
Si on ne sait pas de quel problème il s'agit, je propose que nous fermions l'issue et en ouvrions une autre quand on saura
